.changes/v3.0.0/563-deprecations.md

```diff
@@ -0,0 +1 @@
+* Removed deprecated attributes `ip, network_name, vapp_network_name, network_href, mac, initscript` from `vcd_vapp_vm` [GH-563]
```

Comment:
This information needs to appear in the documentation in that section which @dataclouder added in one of the PRs about removed resources. We need to list these removals in the docs for at least one release of 3.0. We will remove it in further releases.

Reply:
this removals are fields. Ar you sure?

Reply:
Yes. Maybe only section header needs to change (I'm not sure what it's called right now), but we need to state all removals of 3.0 in a single doc place.

Reply:
added

Reply:
Thanks, careful with conflict resolution when you merge ;)